core/consensus/beefy/impl/beefy_impl.hpp

```diff
@@ -18,6 +18,7 @@
 #include "primitives/justification.hpp"
 #include "storage/buffer_map_types.hpp"
 
+#include "network/synchronizer.hpp"
```

Comment:
You forward declared `class Synchronizer;` below,
so `#include` should be in ".cpp" file
```suggestion
```

Reply:
Done